src/ui/marker.js

```diff
@@ -28,6 +29,7 @@ type Options = {
 };
 
 export const TERRAIN_OCCLUDED_OPACITY = 0.2;
+export const MAX_PITCH = 80; // Ensure that markers on globe doen't disappear completely (as they would at pitch 90)
```

Comment:
Unused?
```suggestion
```

Reply:
Good catch! I forgot to remove that when I disabled 3d vertical alignment , it doesn't make much sense when draping.